path/src/run.ps1

```
@@ -0,0 +1,72 @@
+	if (Test-Path $configPath) {
+		try {
+			$fountConfig = Get-Content $configPath -Raw -Encoding UTF8 | ConvertFrom-Json
+			$heapSizeBytes = $fountConfig.prelaunch.heapSize
+			$calculatedMB = [math]::Round($heapSizeBytes / 1024 / 1024)
+			if ($calculatedMB -gt 0) {
+				$heapSizeMB = $calculatedMB
+			}
+		}
+		catch {
+			# Could not read or parse, will use the default 100MB.
+		}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)